### DISCUSSION TITLE
Wakaati TV

### DISCUSSION THREAD

**Discussion by booziebatsymbol** [iptv-org/community#71] [Show and tell]
### Channel ID

WakaatiTV.ng

### Stream URL

https://www.wakaati.com/live

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
https://mn-nl.mncdn.com/wakaatitvsrt/wakaatitvsrt.stream/playlist.m3u8

**Comment by Alstruit** [comment]
Link is 404'd.
